branchpro/models.py

```diff
@@ -42,3 +43,106 @@ def simulate(self, parameters, times):
             value appears in ``times``.
         """
         raise NotImplementedError
+
+#
+# BranchProModel Class
+#
+
+
+class BranchProModel(ForwardModel):
+    """BranchProModel Class:
+    Class for the models following a Branching Processes behaviour.
+    It inherits from the `ForwardModel`` class.
+
+    Parameters
+    ----------
+    initial_r
+        (numeric) Value of the recombination number at the beginning
+        of the epidemic
+    serial_interval:
+        (list) Unnormalised probability distribution of that the recipient
```

Comment:
This is not quite clear, but I also don't have a really good solution from the top of my head. Maybe something like:

List of spreading probabilities for days following the infection.

In any case, a more detailed description of how the model works should be included in the docstring above (where indicated).